.dockerignore

```diff
@@ -137,7 +137,10 @@ dmypy.json
 # datasets
 datasets/shapenetcore_partanno_segmentation_benchmark_v0/
 datasets/lsun
+datasets/cifar10
```

Comment:
I already added `datasets/cifar10` and `benchmarks/mobilenet/` in the main branch. You just need to add `benchmarks/resnet/` and resolve the conflicnts on `.dockerignore` and `.gitignore`.